plugins/de.cau.cs.kieler.klighd.kgraph/pom.xml

```diff
@@ -40,7 +40,6 @@
     <dependency>
       <groupId>org.eclipse.emf</groupId>
       <artifactId>org.eclipse.emf.ecore</artifactId>
-      <version>${emf-ecore-version}</version>
```

Comment:
Some of the dependencies (such as this one) are only handled in the main pom.xml file because otherwise some part of the automatic transitive dependency management breaks - it is marked to be removed in the main pom.xml once the issue no longer persists. Therefore I would like to not remove the version numbering here, because that could cause issues in the future when removing that again.

Alternatively, we could move all version management to one central position in general for consistent versions throughout the build.